libs/filabridge/include/private/filament/UibGenerator.h

```diff
@@ -51,7 +51,9 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     filament::math::mat4f viewFromClipMatrix;
     filament::math::mat4f clipFromWorldMatrix;
     filament::math::mat4f worldFromClipMatrix;
-    filament::math::mat4f lightFromWorldMatrix;
+    filament::math::mat4f lightFromWorldMatrix[CONFIG_MAX_SHADOW_CASCADES];
```

Comment:
this is definitely a material breakage, so need to update the material version

Reply:
Whoops, meant to update it!